src/Groups/GAPGroups.jl

```diff
@@ -216,27 +216,126 @@ function rand_pseudo(G::GAPGroup; radius::Int = 10)
   return group_element(G, GAP.Globals.PseudoRandom(GapObj(G); radius = radius)::GapObj)
 end
 
+
+# We allow arithmetic operations between two group elements with
+# *nonidentical parents*.
+# In this case, the parent of the resulting element is set according to
+# the following rules, depending on the types of the parents.
+#
+# - `PermGroup`, `PermGroup`:
+#   The operation is allowed whenever the parents have the same degree,
+#   and the parent is the symmetric group of that degree.
+#
+# - `PcGroup`, `PcGroup` and
+#   `FPGroup`, `FPGroup`:
+#   The operation is allowed whenever the parents belong to the same `GapObj`
```

Comment:
I don't understand this. When does a `PcGroup` group "belong" to a `GapObj`?

Reply:
O.k., let us say "have the same `GapObj`".